dali/test/python/test_operator_preemph.py

```diff
@@ -65,5 +65,6 @@ def perform(preemph_coeff):
         assert isclose(a, b, abs_tol=eps)
 
 
-perform(0.)
-perform(0.5)
+def test_preemphasis():
+    body(0.0)
```

Comment:
```suggestion
   for coef in [0.5, 0.0]:
       yield check_preemphasis_operator, coef
```
I'd also consider adding the shape as an argument